cmd/aggregated-apiserver/app/options/options.go

```diff
@@ -124,6 +129,8 @@ func (o *Options) Config() (*aggregatedapiserver.Config, error) {
 	}
 
 	serverConfig := genericapiserver.NewRecommendedConfig(aggregatedapiserver.Codecs)
+	serverConfig.LongRunningFunc = customLongRunningRequestCheck(sets.NewString("watch", "follow"), sets.NewString())
+	//serverConfig.Config.RequestTimeout = 10 *time.Minute
```

Comment:
Is this a comment or configuration?

Reply:
I'm sorry. Thanks for reminding, Debugging information. I forgot to delete it.